src/x86_64/page.h

```diff
@@ -18,7 +18,7 @@
 #define PAGEMASK           MASK(PAGELOG)
 #define PAGE_FLAGS_MASK    (PAGE_NO_EXEC | PAGEMASK)
 #define PAGE_PROT_FLAGS    (PAGE_NO_EXEC | PAGE_USER | PAGE_WRITABLE)
-#define PAGE_DEV_FLAGS     (PAGE_WRITABLE | PAGE_WRITETHROUGH | PAGE_NO_EXEC)
+#define PAGE_DEV_FLAGS     (PAGE_WRITABLE | PAGE_CACHE_DISABLE | PAGE_NO_EXEC)
```

Comment:
this was a total rider....I thought cache disable was pretty mandatory for devices...that would imply write through...but its probably academic since there isn't actually any hardware back there?